package/MDAnalysis/core/groups.py

```diff
@@ -3262,16 +3262,16 @@ def split(self, level):
         return [self[levelindices == index] for index in
                 unique_int_1d(levelindices)]
 
-    def guess_bonds(self, vdwradii=None):
+    def guess_bonds(self, vdwradii=None, context='default'):
         """Guess bonds that exist within this :class:`AtomGroup` and add them to
         the underlying :attr:`~AtomGroup.universe`.
 
         Parameters
         ----------
         vdwradii : dict, optional
-            Dict relating atom types: vdw radii
-
-
+        Dict relating atom types: vdw radii
+        context : string
+        context of the universe
```

Comment:
```suggestion
            Dict relating atom types: vdw radii
        context : string
            context of the universe
```
We should stick to the style guide for formatting docs.